internal/cli/atlas/cloudprovider/aws/create.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aws
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+const (
+	provider       = "AWS"
+	createTemplate = "AWS IAM role successfully created.\n"
+)
+
+type CreateOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	store store.AccessRoleCreator
+}
+
+func (opts *CreateOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+func (opts *CreateOpts) Run() error {
+	r, err := opts.store.AccessRole(opts.ConfigProjectID(), provider)
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+// mongocli atlas cloudProvider aws accessRoles create [--projectId projectId]
```

Comment:
it looks like your implementation is 
`mongocli atlas cloudProvider aws create`

we are missing `accessRoles`?

Also looking at the API and given we can't filter the list by provider we may want to do this 
`mongocli atlas cloudProvider accessRoles aws create`, I may even give it to you and join it back `cloudProviderAccessRoles` but it's just such along command, cc @themantissa if she has suggestions here

Reply:
oh right, I missed `accessRoles`.

Reply:
No suggestions from me.  Not sure how you could shorten it much more sadly.

Reply:
I moved commands around and now we have `mongocli atlas cloudProvider accessRoles aws create` so that we can have `mongocli atlas cloudProvider accessRoles list`